jerry-debugger/src/lib/cdt-proxy.ts

```diff
@@ -70,14 +69,28 @@ export class ChromeDevToolsProxyServer {
       logConsole: true,
     });
 
-    // Lifted from the example on https://github.com/nojvek/noice-json-rpc
-    const enable = async () => {
-      console.log('enable called!');
+    // Based on the example from https://github.com/nojvek/noice-json-rpc
+    const notImplemented = async () => {
+      console.log('Function not implemented');
     };
 
-    api.Debugger.expose({ enable });
+    // QUESTION: is there a better way to do this?
+    const proxy = this;
```

Comment:
@martijnthe I'm interested to see what you say about this? I'm running into this pattern again elsewhere right now.

It would seem I could also add a dynamic member to the other object below, but I'm not sure TypeScript likes that. So then maybe the option would be to inherit from the class and store the proxy object in a new member that way.

Reply:
Arrow functions are invented for this (no pun intended).
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions